components/omega/src/timeStepping/RungeKutta4Stepper.cpp

```diff
@@ -47,13 +47,10 @@ void RungeKutta4Stepper::doStep(OceanState *State, TimeInstant Time) const {
 
    const int CurLevel  = 0;
    const int NextLevel = 1;
```

Comment:
My two cents is that in this case the number values have clear meanings and are documented. The levels can always be named for additional clarity as they are in the timestepping routines.